### PR TITLE
Organise settings interface

### LINKED ISSUE #586: Organise settings interface
- Allow for grouping and sorting of the settings interface
- Each section should have a header and possibly also a description
- Wireframes needed

### PR DESCRIPTION
**Pull request by nebs-dev**
Resolves #586 

- migration for the new settings field `section`
- modified types and transformer
- created separate components for each setting group

### CODE REVIEW

**Comment by nebs-dev** [comment]
@kristoferlund can you take a look at this? 
I separated each setting group into components. I know components are almost the same but i though it is more clean this way and also maybe sometimes in future we would like to modify these screens more
Also i added DB field because maybe we will add option to move fields from one section into other and than we only need to change this new `section` value

File: packages/frontend/src/pages/Settings/components/SettingsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+export const SettingsSection = ({
+  header,
+  description,
+  children,
+}: Props): JSX.Element => {
+  return (
+    <div>
+      <div className="mb-8">
```
Comment (kristoferlund) [inline packages/frontend/src/pages/Settings/components/SettingsSection.tsx:14]: Margin 5 is the "default" margin we use in most places.

File: packages/frontend/src/pages/Settings/components/SettingsSection.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <div className="mb-2 text-xl">{header}</div>
```
Comment (kristoferlund) [inline packages/frontend/src/pages/Settings/components/SettingsSection.tsx:15]: Use `<h2>`

**Review by kristoferlund — CHANGES_REQUESTED** [review]
Nice!

I agree with breaking out each settings group into components, looks good! 

Made some small styling comments.

One thing I think you should change though. We use the term `group` and `SettingsGroup`. `section` is very closely related to the groups, I think `subgroup` would more clearly show the connection.

**Review by kristoferlund — APPROVED** [review]
Nice Nebs! Merging!

**Pull request merged**